pkg/strategy/strategy.go

```diff
@@ -32,5 +32,5 @@ type Strategy struct {
 	HorizontalPodAutoscalers []autoscalingv2beta2.HorizontalPodAutoscaler
 	Services                 []corev1.Service
 	Secrets                  []corev1.Secret
-	Collector                *otelv1alpha1.OpenTelemetryCollector
+	Collectors               []*otelv1alpha1.OpenTelemetryCollector
```

Comment:
Might be worth considering renaming this to OtelCol to avoid confusion with Jaeger Collector.

Reply:
Totally agree